code/__HELPERS/text.dm

```diff
@@ -63,6 +63,10 @@
 	var/regex/alphanum_only = regex("\[^a-zA-Z0-9# ,.?!:;()]", "g")
 	return alphanum_only.Replace(t, "#")
 
+/proc/multiline_paranoid_sanitize(txt)
+	var/regex/alphanum_only = regex("\[^a-zA-Z0-9# ,.?!:;\\r\\n()]", "g")

```

Comment:
This actually doesn't work and will still filter newlines. Could anyone offer a solution?